pkg/cli/admin/release/mirror.go

```diff
@@ -890,14 +897,41 @@ func printImageContentInstructions(out io.Writer, from string, toList []string,
 	sources = uniqueSources
 
 	// Create and display install-config.yaml example
+	v, err := semver.Parse(version)
+	if err != nil {
+		return fmt.Errorf("unable to parse version '%s': %v", version, err)
+	}
+	if v.Major >= 4 && v.Minor >= 14 {
```

Comment:
We are adding this temporarily. Could you please add a TODO comment pointing that this conditional should be removed when icsp support is out of sight?